tensorflow/examples/tutorials/mnist/fully_connected_feed.py

```diff
@@ -251,9 +251,9 @@ def main(_):
       help='Batch size.  Must divide evenly into the dataset sizes.'
   )
   parser.add_argument(
-      '--train_dir',
+      '--data_dir',
```

Comment:
I get what you saying here, maybe we need to create another directory which storage all the graph and checkpoint by the model name and type
 for example
/tmp/data/mnist/input_data
/tmp/data/mnist/output_data
/tmp/data/imagenet/input_data
/tmp/data/imagenet/output_data
so that when we run different model in a day we don't need to remove the other graph file to make it clean